ai_flow/frontend/web_server.py

```diff
@@ -335,6 +336,11 @@ def graph(project_id, workflow_name):
     return render_template('index.html')
 
 
+@app.route('/version')
```

Comment:
Should we add a simple test case to test this simple method? Just don't want to see the code coverage drop.